src/analysis/special_functions/gamma/bohr_mollerup.lean

```diff
@@ -377,4 +379,91 @@ end
 
 end strict_mono
 
+section doubling
+
+/-!
+## The Gamma doubling formula
+
+As a fun application of the Bohr-Mollerup theorem, we prove the Gamma-function doubling formula
+(for positive real `s`). The idea is that `2 ^ s * Gamma (s / 2) * Gamma (s / 2 + 1 / 2)` is
+log-convex and satisfies the Gamma functional equation, so it must actually be a constant
+multiple of `Gamma`, and we can compute the constant by specialising at `s = 1`. -/
+
```

Comment:
Can't the same cute proof give the general multiplication formula for `Gamma(z) Gamma(z+1/k) ... Gamma(z + (k-1)/k)` (modulo Stirling formula to identify the constant, at infinity)? If so, it might be worth mentioning it here and adding a todo.

Reply:
Indeed. The only thing that is genuinely easier for `k=2` is the identification of the constant. I'll add that as a `todo`, it might be a fun student project 🙂